### ISSUE TITLE
how to fine tune bert for ner on custom data?

### ISSUE THREAD

**Issue by Anticsss** [google-research/bert#713]
I want to fine tune bert on my custom dataset but i don't know how to tag sentences and do fine tuning if anyone knows please suggest me.

**Comment by sofieneJ** [comment]
Hello,
I am looking for the same thing.
There is this link to a colab notebook : https://colab.research.google.com/github/tensorflow/tpu/blob/master/tools/colab/bert_finetuning_with_cloud_tpus.ipynb
But it is restricted to MRPC, CoLA, MNLI tasks for which you need an appropriate formated dataset from your corpus.
I would have expected to be able to fine -tune on my training corpus (unlabeled) using masked language model task.
Have you found smthg else ?

**Comment by inessoveral** [comment]
@Anticsss @sofieneJ I am also looking for this. Did any of you manage to find a solution?

**Comment by Ankan1998** [comment]
I was also trying to use BERT for custom NER. I found a slotion kind of. Check out
https://www.kaggle.com/amrkeleg/fine-tuning-bert-models-for-bio-entity-recognition
used bert for biological NER.

**Comment by ghost** [comment]
@Ankan1998 what is this? is this a building dataset format for bert?
\

**Comment by Sanjiv-Thapa** [comment]
i also need the same solution do anyone help out
